apps/omg_db/lib/omg_db/release_tasks/init_keys_with_values.ex

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.DB.ReleaseTasks.InitKeysWithValues do
```

Comment:
@InoMurko what do you think about this solution?
It uses a release task to set a value if it's not already set.
I am not sure we should use a config provider here as this does not change config at all.

Reply:
Why not take the same approach as `OMG.WatcherInfo.ReleaseTasks.InitPostgresqlDB`? Why use https://github.com/omgnetwork/elixir-omg/pull/1720/files#diff-3cc6d83c8ce1f76e93166823f0298352R19 a config provider?

Reply:
Thanks, that's exactly what I needed.